feature/xhr/aggregate/index.js

```diff
@@ -24,7 +24,8 @@ var sentAjaxEvents = []
 // bail if not instrumented
 if (!loader.features.xhr) return
 
-var harvestTimeSeconds = config.getConfiguration('ajax.harvestTimeSeconds') || 60
+var harvestTimeSeconds = config.getConfiguration('ajax.harvestTimeSeconds') || 10
+var MAX_PAYLOAD_SIZE = config.getConfiguration('ajax.maxPayloadSize') || 500
```

Comment:
Should the default be 1MB?

Reply:
yes thanks, forgot to set this back after i was manually testing it